bim2sim/tasks/bps/enrich_material.py

```diff
@@ -180,7 +183,7 @@ def get_material_template(cls, material_name: str, resumed_names: list,
                                                                material_name)
         return resumed[selected_material]
 
-    def enrich_layer_set(self, invalid_element, resumed, templates):
+    def enrich_layer_set(self, invalid_element: Any, resumed: dict, templates: dict):
```

Comment:
I think we will get different types if we run the different cases, so I just set it to "Any".
![image](https://github.com/BIM2SIM/bim2sim/assets/159027366/fb995d86-da80-4023-bb72-23cb6cd178a5)

Reply:
Function doesn't exist anymore, so I guess this is obsolete